eox-core-v-1-0/prose/edit/src/design-considerations-00.md

```diff
@@ -1,13 +1,67 @@
 # Design Considerations
 
-OpenEoX is a language to exchange any End-of information formulated in JSON.
+The OpenEoX framework is a comprehensive language to exchange End-of-X (EoX) lifecycle information
```

Comment:
I think what we want to say (ignoring the f-word and removing ratings the reader should provide) is more like:

>OpenEoX provides the vocabulary for exchanging standardized End-of-X (EoX) life cycle data for hardware and software, as well as for all supplier-consumer relationships.